substrate/frame/transaction-payment/src/payment.rs

```diff
@@ -191,4 +197,14 @@ where
 		}
 		Ok(())
 	}
+
+	#[cfg(feature = "runtime-benchmarks")]
+	fn endow_account(who: &T::AccountId, amount: Self::Balance) {
+		let _ = C::deposit_creating(who, amount);
```

Comment:
```suggestion
		assert!(C::deposit_creating(who, amount).peek() != amount, "Could not endow account.");
```
The benchmark tests actually fail like this 🤔

Reply:
Isn't it supposed to be `assert!(C::deposit_creating(who, amount).peek() == amount, "Could not endow account.");` in your example? As far as I understand, if `deposit_creating` succeeded, the returned positive imbalance would be equal to the `amount` parameter, as in the entire amount was minted successfully.

Reply:
Oh yea, i was playing around with the code before. 👍